Pi_Hole_Ad_Blocker/stats.py

```diff
@@ -96,18 +96,24 @@
     except:
       time.sleep(1)
       continue
+        
+#DirtyDaveMcGee edit to improve readability
+#192.168.x.x addresses were being cut off on my display
+#and i wanted to keep the cpu, disk, and memory stats
 
-    draw.text((x, top),       "IP: " + str(IP) + "( " + HOST + ")",  font=font, fill=255)
+    draw.text((x, top),	      str(HOST), font=font, fill=255)
     draw.text((x, top+8),     "Ads Blocked: " + str(ADSBLOCKED), font=font, fill=255)
     draw.text((x, top+16),    "Clients:     " + str(CLIENTS),  font=font, fill=255)
     draw.text((x, top+24),    "DNS Queries: " + str(DNSQUERIES),  font=font, fill=255)
```

Comment:
Could you change the above to use `.format()` or '%' instead of string concatenation?

Reply:
If I had the slightest clue as to what I was doing, I'd guess it was possible. This is a copy paste hack based on my non-understanding of python. It works for me. I believe the code was in a tutorial on adafruit.com